Polindrom/script.js

```diff
@@ -0,0 +1,19 @@
+function palindrome(str) {
```

Comment:
![image](https://user-images.githubusercontent.com/29356908/144327435-7d4491d4-6369-4b63-adcb-3a9af7b76547.png)

Reply:
rename the branch and add numbers for regex please

Reply:
I have noticed it yesterday,  i thought to fix name`s branch this morning, but you have seen the wrong branch early then i repair it